Foundation/NSByteCountFormatter.swift

```diff
@@ -40,49 +40,57 @@ public enum NSByteCountFormatterCountStyle : Int {
 }
 
 public class NSByteCountFormatter : NSFormatter {
+    public override init() {
+        super.init()
+    }
+    
     public required init?(coder: NSCoder) {
         NSUnimplemented()
     }
     
     /* Shortcut for converting a byte count into a string without creating an NSByteCountFormatter and an NSNumber. If you need to specify options other than countStyle, create an instance of NSByteCountFormatter first.
     */
-    public class func stringFromByteCount(byteCount: Int64, countStyle: NSByteCountFormatterCountStyle) -> String { NSUnimplemented() }
-    
+    public class func stringFromByteCount(byteCount: Int64, countStyle: NSByteCountFormatterCountStyle) -> String {
+        let byteCountFormatter = NSByteCountFormatter()
```

Comment:
It just calls into a private function which is shared with the formatter object.